examples/spark/src/main/scala/org/apache/carbondata/examples/CDCExample.scala

```diff
@@ -141,6 +141,8 @@ object CDCExample {
         .write
         .format("carbondata")
         .option("tableName", "target")
+        .option("sort_scope", "global_sort")
```

Comment:
I see the same problem for local sort also, 
**So change PR name.**

Reply:
Changed